api/polling/README.md

```diff
@@ -1,60 +1,127 @@
+#### Order processing tips
 
-Use the [Poll shipments](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/shipment/specs.yaml#/shipment/pollShipmentsRoute) endpoint for polling shipments.
+When working with order endpoints:
 
-## Step 2. Process the orders or shipments in the search response body
+- Use `data.lines.lastUpdatedAt` to identify which specific lines changed
+- Filter by status using `data.lines.status` fields:
+  - `processStatus`
+  - `inProgressStatus`
+  - `logisticsStatus`
+  - `deliveryLineStatus`
 
-Process the fetched new or updated orders or shipments.
+#### Shipment processing tips
 
-{% hint style="warning" %}
-You may receive any order or shipment change, including echoed changes from your ERP system. How to handle your own open request is explained in [Polling echo](echo.md).
-{% endhint %}
+When working with shipment endpoint:
 
-When using the [Poll orders](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/order-search/specs.yaml#/order-search/pollOrdersRoute) or [Poll single delivery orders](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/order-search/specs.yaml#/order-search/pollOrdersSingleDeliveryRoute) endpoints:
+- Use `data.lines.meta.lastUpdatedAt` to identify changed shipment lines
+- Filter by `data.status` to focus on relevant shipment statuses
 
-* Optionally use the `data.lines.lastUpdatedAt` field to filter the order lines that have been changed: Only consider lines where `lines.lastUpdatedAt` is after the `lastUpdatedAfter` filter value.
-* Optionally use the `data.lines.status` fields to filter the order lines on `processStatus`, `inProgressStatus`, `logisticsStatus` and `deliveryLineStatus` fields.
+### Step 3: Persist the timestamp
 
-When using the [Poll shipments](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/shipment/specs.yaml#/shipment/pollShipmentsRoute) endpoint:
+Store the `lastUpdatedAt` value to use as `lastUpdatedAfter` in subsequent requests.
 
-* Optionally use the `data.lines.meta.lastUpdatedAt` to filter the shipment lines that have been changed.
-* Optionally use the `data.status` field to filter on shipment process and logistics status sub fields.
+#### Implementation tips
 
-## Step 3. Persist the `lastUpdatedAt` for the next polling request
+- The timestamp has format `YYYY-MM-DDThh:mm:ss.SSSZ` (ISO 8601)
+- Store this in **persistent storage** (database, file system, etc.)
+- Your storage solution must survive application restarts or crashes
+- If the response contains no data, reuse your current `lastUpdatedAfter` value
+- For first-time polling, use a historical date to fetch existing records
 
-Persist the **latest** \(in the last order or shipment in the response body\) `lastUpdatedAt` to be used as `lastUpdatedAfter` in the next polling request.
+#### Edge cases
```

Comment:
What does this section mean?
I don't understand the first bullet. The second bullet is already mentioned directly above this header.